ee/frontend/mobile-replay/mobile.types.ts

```diff
@@ -0,0 +1,155 @@
+// copied from rrweb-snapshot, not included in rrweb types
```

Comment:
`rrweb` is MIT, should we copy-paste the license and credits as a header? not an expert but https://www.tldrlegal.com/license/mit-license

Reply:
We've got that package installed as well, so I think we're ok on MIT license front. And I copied them here because it's weird there's a types package and then essential types that aren't in it. I want an aide memoire that we should PR to move those types into the types package